nvmf-autoconnect/udev-rules/71-nvmf-iopolicy-netapp.rules.in

```diff
@@ -1,3 +1,6 @@
 # Enable round-robin for NetApp ONTAP and NetApp E-Series
 ACTION=="add", SUBSYSTEM=="nvme-subsystem", ATTR{subsystype}=="nvm", ATTR{model}=="NetApp ONTAP Controller", ATTR{iopolicy}="round-robin"
 ACTION=="add", SUBSYSTEM=="nvme-subsystem", ATTR{subsystype}=="nvm", ATTR{model}=="NetApp E-Series", ATTR{iopolicy}="round-robin"
+
+# Set ctrl_loss_tmo to -1 for NetApp ONTAP NVMe/TCP
+ACTION=="add|change", SUBSYSTEM=="nvme", KERNEL=="nvme*", ATTR{transport}=="tcp", ATTR{model}=="NetApp ONTAP Controller", ATTR{ctrl_loss_tmo}="-1"
```

Comment:
IIRC, the recommended `ACTION` match is to use `ACTION!=remove` instead.

Reply:
Ok. Will make that change.